api/exceptions.inc

```diff
@@ -100,15 +100,15 @@ class ForbiddenError extends ApiException
 
 class UnexpectedError extends ApiException
 {
-    public function __construct($message = "Unexpected error", $code = 7)
+    public function __construct(string $message = "Unexpected error", int $code = 7)
```

Comment:
I didn't fix this class using the same `$code` as the `ForbiddenError` (nor the general fact that `$code` is rarely updated and could be a class constant).

Reply:
Is this something that should be fixed, or does it not matter?

Reply:
Both issues are very minor and I don't believe they cause any issue.

Reusing `$code` would be a bother if we did have some dashboard for the exception thrown in prod/test grouped by `$code` as we would misclassify some of them.

Not having `$code` be a constant is more of a design/code quality point so pretty low on the totem pole.